webpack/envs/clientCommonConfig.js

```diff
@@ -135,6 +131,11 @@ export const clientCommonConfig = {
     // Symlink issues should be fixed via `yarn --pnp`
     modules: [path.resolve(basePath, "src"), "node_modules"],
     symlinks: false,
+    fallback: {
+      "path": false,
+      "os": require.resolve("os-browserify/browser"),
+      "buffer": require.resolve("buffer/"),
```

Comment:
What does this section + individual keys do?

Reply:
Webpack 5 no longer provides automatic browser polyfills. This section specifies the recommended polyfill to use for each node api we access in the browser.